<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransferFunctionWidget</class>
 <widget class="QWidget" name="TransferFunctionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>115</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout"/>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox_colorScheme">
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>20</height>
      </size>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rbtn_Continuous">
     <property name="text">
      <string>Continuous</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rbtn_BandedMidpoint">
     <property name="text">
      <string>Banded Midpoint</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rbtn_BandedSides">
     <property name="text">
      <string>Banded Sides</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
